code/game/gamemodes/cult/runes.dm

```diff
@@ -711,7 +711,7 @@
 			T.imbue = "emp"
 			imbued_from = R
 			break
-		if(R.word1==cultwords["blood"] && R.word2==cultwords["see"] && R.word3==cultwords["destroy"]) //conceal
+		if(R.word1==cultwords["hide"] && R.word2==cultwords["see"] && R.word3==cultwords["blood"]) //conceal
```

Comment:
He doesn't like the lack of defines

Also something something rune datums; flat comparisons of listed strings

Not your fault